modules/mobile/docs/examples/appointments/cc_appointment.yml

```diff
@@ -41,11 +41,6 @@ data:
       patient_email: null
       best_time_to_call: null
       friendly_location_name: null
-      locationId: '938'
```

Comment:
Location id is not in our serializer. I think this may have been added when we were working on appointment scheduling, but it doesn't seem to apply to the index.